app/Community/Requests/UpdateEmailRequest.php

```diff
@@ -17,7 +17,7 @@ public function authorize(): bool
     public function rules(): array
     {
         return [
-            'newEmail' => 'required|email',
+            'newEmail' => 'required|email|disposable_email',
```

Comment:
Would you be opposed to changing this to `not_disposable_email`? It reads like the email must be disposable to be allowed.

Reply:
I agree 100%. `disposable_email` is what's exported by the library itself. The only idea I have is to make an alias for the rule name, which I've done in latest.